subprojects/docs/src/docs/userguide/authoring-builds/basics/writing_build_scripts.adoc

```diff
@@ -0,0 +1,557 @@
+
+image::author-gradle-6.png[]
+
+Then, for each project included in the settings file, Gradle creates a link:{javadocPath}org/gradle/api/Project.html[`Project`] instance.
+
+Gradle then looks for a corresponding build script file, which is used in the configuration phase.
+
+[[tutorial_using_tasks]]
+== Build Scripts
+
+Every Gradle build comprises one or more *projects*; a _root_ project and _subprojects_.
+
+A project typically corresponds to a software component that needs to be built, like a library or an application.
+It might represent a library JAR, a web application, or a distribution ZIP assembled from the JARs produced by other projects.
+
+On the other hand, it might represent a thing to be done, such as deploying your application to staging or production environments.
+
+Gradle scripts are written in either Groovy DSL or Kotlin DSL (domain-specific language).
+
+A *build script* configures a *project* and is associated with an object of type link:{javadocPath}org/gradle/api/Project.html[`Project`].
+
+image::author-gradle-4.png[Build,500,63,align="left"]
+
+As the build script executes, it configures `Project`.
+
+The build script is either a `\*.gradle` file in Groovy or a `*.gradle.kts` file in Kotlin.
+
+IMPORTANT: _Build scripts_ define `Project` objects.
+
+== The `Project` object
+
+The `Project` object is part of the link:{javadocPath}org/gradle/api/Project.html[Gradle API].
+
+- In the Groovy DSL, the `Project` object documentation is found link:{groovyDslPath}/org.gradle.api.Project.html[here].
+- In the Kotlin DSL, the `Project` object documentation is found link:{kotlinDslPath}/org.gradle.api.initialization/-settings/index.html[here].
+
+Many top-level properties and blocks in a build script are part of the Project API.
+
+For example, the following build script uses the link:{groovyDslPath}/org.gradle.api.Project.html#org.gradle.api.Project:name[Project.name] property to print the name of the project:
+
+====
+include::sample[dir="snippets/tutorial/projectApi/kotlin",files="build.gradle.kts[tags=project-name]"]
+include::sample[dir="snippets/tutorial/projectApi/groovy",files="build.gradle[tags=project-name]"]
+----
+$ gradle -q check
+include::{snippetsPath}/tutorial/projectApi/tests/projectApi.out[]
+----
+====
+
+Both `println` statements print out the same property.
+
+The first uses the top-level reference to the `name` property of the `Project` object.
+The second statement uses the `project` property available to any build script, which returns the associated `Project` object.
+
+[[sec:standard_project_properties]]
+=== Standard project properties
+
+The `Project` object exposes a standard set of properties in your build script.
+
+The following table lists a few commonly used properties:
+
+[%header%autowidth,compact]
+|===
+| Name | Type | Default Value
```

Comment:
The last column heading is misused. In some rows it is the default value, others it is the description of the property.